agot-bg-game-server/src/client/GameLogListComponent.tsx

```diff
@@ -83,7 +83,7 @@ export default class GameLogListComponent extends Component<GameLogListComponent
                 return (
                     <>
                         <b>{attacker.name}</b> attacked <b>{attacked ? attacked.name : "a neutral force"}</b> from <b>{attackingRegion.name}</b> to <b>
-                        {attackedRegion.name}</b> with {joinReactNodes(army.map(ut => ut.name), ', ')}.
+                        {attackedRegion.name}</b> with <strong>{joinReactNodes(army.map(ut => ut.name), ', ')}</strong>.
```

Comment:
Put the `<strong>` inside the `joinReactNodes` so that the commas are not bolded.